include/z64scene.h

```diff
@@ -271,7 +271,7 @@ typedef struct {
 } Room; // size = 0x14
 
 typedef struct {
-    /* 0x00 */ Room currRoom;
+    /* 0x00 */ Room curRoom;
```

Comment:
The only reason this was touched was because `curRoom` was used in some of the decomp code here, specifically pertaining the minimap hud icon. This was more of a typo/styling fix than an actual doc, since `cur` is the style we go for instead of `curr`.

I could see arguments to name this `curRoom` (since there's an opposing previous room), but also `room`, `roomId`, and `roomIndex` all seem like good candidates. Since `z_room.c` is still only half documented, I think that decomp/PR would be a good place to have that discussion, and just leave this to a typo fix for now?

Reply:
I think we should leave that discussion for a future `z_room.c` docs pass